### PR TITLE
Task: Show Balances & Redux

### PR DESCRIPTION
**Pull request by martink-rsa**
Task: Show Balances:

1. Removed the abstracted DisplayCurrency components since they function differently; one has an external link and the other has a modal. It required a lot of customization and complexity to deal with these two different cases so it made sense to remove them as a reusable component. They could be added a standalone component if we end up with too many lines.
2. Added redux with only a `setNet` action that we use on the initial load of the component. This lets us use `net` throughout the app.

### CODE REVIEW

File: src/react-hooks/useNet.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from 'react';
```
Comment (jakai7) [inline src/react-hooks/useNet.tsx]: What is this file (useNet) for? What does it fetch?

Comment (martink-rsa) [reply src/react-hooks/useNet.tsx]: This was just boilerplate for a hook I was considering to use the `net` functionality throughout the app before moving to Redux. I thought it had been deleted though. Double-checking why it's still around

**Comment by jakai7** [comment]
> @martink-rsa commented on this pull request.
>
>
>
>> @@ -0,0 +1,23 @@
> +import React, { useState, useEffect } from 'react';
>
> This was just boilerplate for a hook I was considering to use the
`net` functionality throughout the app before moving to Redux. I
thought it had been deleted though. Double-checking why it's still
around

Okay

**Comment by jakai7** [comment]
I don't know Redux to review it, everything else looks good :-)

**Review by jakai7 — APPROVED** [review]

**Pull request merged**